packages/framework/src/parser.ts

```diff
@@ -108,7 +117,11 @@ export const EnsembleParser = {
       screens: screens as EnsembleScreenModel[],
       customWidgets,
       home: menu ?? screens[0],
-      theme,
+      themes: themes
+        ? (Object.assign({}, ...themes) as {
+            [key: string]: EnsembleThemeModel;
+          })
+        : undefined,
```

Comment:
This might be why your themes are blanking out - make sure the `unwrapTheme` is not mutating the same object and causing this to be empty.